### ISSUE TITLE
Previewing Sounds is Frustrating on Filterblade

### ISSUE THREAD

**Issue by livejamie** [NeverSinkDev/NeverSink-Filter#88]
It would be nice to easily preview the soundpacks that are available but after a soundpack is selected you can't go back into the presets. You have to reset and start all over.

**Comment by Tobnac** [comment]
Thanks for the input, we'll see how we can improve this.

I personally always used the sound selection UI to check out all the sounds. Using the dropDown menus with the arrow keys was completely fine for me.

**Comment by NeverSinkDev** [comment]
Yeah, it'd be nice to improve on that, sadly this was somewhat very hard to implement at the beginning, due to timelimits and some architecture concerns. We'll see what we can do.
